formbuilder/components/builder/AddFieldDropdown.js

```diff
@@ -8,12 +8,12 @@ import config from "../../config";
 export default class AddFieldDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {'fieldList': config.fieldList};
+    this.state = {"fieldList": config.fieldList};
   }
 
   handleAddField(field_index, event) {
```

Comment:
`field_index` should be named `fieldIndex` in camel case, in JavaScript fashion.